pkg/cmd/cli/cmd/status.go

```diff
@@ -61,3 +79,30 @@ func RunStatus(f *clientcmd.Factory, out io.Writer) error {
 	fmt.Fprintf(out, s)
 	return nil
 }
+
+// RunGraph contains all the necessary functionality for the OpenShift cli graph command
+func RunGraph(f *clientcmd.Factory, out io.Writer) error {
+	client, kclient, err := f.Clients()
+	if err != nil {
+		return err
+	}
+
+	namespace, _, err := f.DefaultNamespace()
+	if err != nil {
+		return err
+	}
+
+	describer := &describe.ProjectStatusDescriber{kclient, client}
+	g, err := describer.MakeGraph(namespace)
+	if err != nil {
+		return err
+	}
+
+	data, err := dot.Marshal(g, namespace, "", "  ", false)
```

Comment:
> Can you wrap this in a g.DOT() method? I was planning on implementing it for build-chain. You won't need to re-import encoding/dot here too.

I see this as equivalent to our JSON and YAML encoders.  We don't provide helper methods for those serializations either.